modules/runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/log/TestLogChronicle.java

```diff
@@ -74,6 +76,10 @@ public LogManager createManager() throws Exception {
 
     @Test
     public void deleteInvalidPath() throws Exception {
+        if (IS_WIN) {
+            // HACK for Windows JUnit test
+            return;
+        }
```

Comment:
Please use just `assumeFalse(SystemUtils.IS_OS_WINDOWS)` instead.

Reply:
assumeFalse(IS_WIN) preferred to keep independency of nuxeo-stream from commons-lang3